packages/language-server/test/plugins/typescript/features/DiagnosticsProvider.test.ts

```diff
@@ -169,5 +169,5 @@ describe('DiagnosticsProvider', function () {
         assert.deepStrictEqual(diagnostics3.length, 0);
         const diagnostics4 = await plugin.getDiagnostics(otherDocument);
         assert.deepStrictEqual(diagnostics4.length, 0);
-    }).timeout(5000);
+    }).timeout(this.timeout() * 2.5);
```

Comment:
The one with the hard-coded timeout should have something like this. It's mostly tests where typescript will rebuild its type-checking program during the test, so it might take longer than the default timeout in a lot of machines.